eng/source-build-patches/0003-Fix-casing-in-project-files.patch

```diff
@@ -0,0 +1,67 @@
+From ef25c5a4142b1a52c7996e8a281afa879c5ae3d0 Mon Sep 17 00:00:00 2001
```

Comment:
Should we just apply this patch to the repo itself? It should cause any problems.

Reply:
The ArPow (arcade-powered source-build) plan calls for `.patch` files to be checked in at first, then incorporated by the repo champ asynchronously. This makes it so the ArPow effort as a whole is unblocked at the earliest opportunity. https://github.com/dotnet/source-build/blob/master/Documentation/planning/arcade-powered-source-build/implementation-plan.md#stage-descriptions

Since the PR's CI is green, it seems to me that it would be best to merge it, and leave applying this patch as a follow-up item. (Along with incorporating the other patch.)